.vscode/launch.json

```diff
@@ -0,0 +1,177 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Safety Auth Login",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
```

Comment:
We could consider making the python path a variable, or also letting VSCode use the selected interpreter